src/v3/src/transformer/terminal/transformUnhandledErrors.ts

```diff
@@ -52,7 +52,11 @@ const getErrorMessage = (error?: AuthApiError, widgetProps?: WidgetProps) : stri
       message: () => loc('oie.feature.disabled', 'login'),
     },
     {
-      tester: (err?: AuthApiError) => err?.errorCode && !!err?.errorSummary,
+      tester: (err?: AuthApiError) => err?.errorCode && err?.errorSummary,
+      message: (err?: AuthApiError) => err?.errorSummary,
+    },
+    {
+      tester: (err?: AuthApiError) => err?.errorCode && !err?.errorSummary,
```

Comment:
This diff is important @glenfannin-okta -- I removed a `!` here so instead of testing for the existence of `errorSummary`, we test for the non-existence of `errorSummary`. In the case where the _is_ an `errorSummary`, we use the tester above this one to directly render the `errorSummary` as the message in the infobox.

See the logic [here](https://github.com/okta/okta-signin-widget/blob/ff6816c5012387e4cfab4f25f16ba9d386b0570e/src/v2/controllers/FormController.ts#L343-L350) that I am copying. I saw we had unit tests asserting the existing behavior so I want to make sure we didn't intentionally change the behavior for gen3 before I merge this.